### DISCUSSION TITLE
when is it necessary to build carla?

### DISCUSSION THREAD

**Discussion by mjpablo23** [carla-simulator/carla#5151] [Q&A]
I would like to use Carla to run some navigation and path planning algorithms in a custom environment.  I need to get new hardware to run Carla.  Is it necessary for me to build Carla, or can I just download and install it?  There is this link to download it:  
https://github.com/carla-simulator/carla/blob/master/Docs/download.md

When is it necessary to build it?  The requirements to build it seem to be much bigger than to just install it.  

install requirements:
https://carla.readthedocs.io/en/latest/start_quickstart/
An adequate GPU. CARLA aims for realistic simulations, so the server needs at least a 6 GB GPU although we would recommend 8 GB. A dedicated GPU is highly recommended for machine learning.
Disk space. CARLA will use about 20 GB of space.

buid requirements:
https://github.com/carla-simulator/carla
Recommended system
Intel i7 gen 9th - 11th / Intel i9 gen 9th - 11th / AMD ryzen 7 / AMD ryzen 9
+16 GB RAM memory
NVIDIA RTX 2070 / NVIDIA RTX 2080 / NVIDIA RTX 3070, NVIDIA RTX 3080
Ubuntu 18.04

**Comment by Peter-P779** [comment]
It is nessary to build it if you want to use it on a different operting system or linux distribution like CentOS.

**Reply by lkpAgent** [reply]
can you run the carla in centos system?
